{
	"schema_version": "1.6.0",
	"id": "BIT-pillow-2021-25291",
	"modified": "2024-03-06T11:25:28.861Z",
	"published": "2024-03-06T11:04:42.663Z",
	"aliases": [
		"CVE-2021-25291",
		"GHSA-mvg9-xffr-p774",
		"PYSEC-2021-37"
	],
	"details": "An issue was discovered in Pillow before 8.1.1. In TiffDecode.c, there is an out-of-bounds read in TiffreadRGBATile via invalid tile boundaries.",
	"affected": [
		{
			"package": {
				"ecosystem": "Bitnami",
				"name": "pillow",
				"purl": "pkg:bitnami/pillow"
			},
			"severity": [
				{
					"type": "CVSS_V3",
					"score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
				}
			],
			"ranges": [
				{
					"type": "SEMVER",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "8.1.1"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://github.com/bitnami/vulndb/tree/main/data/pillow/BIT-pillow-2021-25291.json"
			}
		}
	],
	"references": [
		{
			"type": "WEB",
			"url": "https://pillow.readthedocs.io/en/stable/releasenotes/8.1.1.html"
		},
		{
			"type": "WEB",
			"url": "https://security.gentoo.org/glsa/202107-33"
		}
	],
	"database_specific": {
		"cpes": [
			"cpe:2.3:a:python:pillow:*:*:*:*:*:*:*:*"
		],
		"severity": "High"
	}
}
